library/std/src/sys/unix/process/process_common.rs

```diff
@@ -429,12 +429,20 @@ impl Stdio {
 }
 
 impl From<AnonPipe> for Stdio {
+    /// Converts a [`AnonPipe`] into a [`Stdio`].
+    ///
+    /// The conversion consumes the [`AnonPipe`].
+    ///
     fn from(pipe: AnonPipe) -> Stdio {
         Stdio::Fd(pipe.into_inner())
     }
 }
 
 impl From<File> for Stdio {
```

Comment:
These two implementations are not exposed outside of std, as Stdio here is an internal type (defined in this very module, and not public). This can be a little confusing since there's also a public std::process::Stdio, but that's an unrelated type.

I don't think it's useful to add documentation to these, particularly since these impls are pretty trivial -- we're not doing anything interesting in them.